pkg/apis/operator/v1alpha1/types.go

```diff
@@ -83,6 +83,9 @@ type Settings struct {
 	// cluster.
 	// +optional
 	LoadBalancerServices *SettingLoadBalancerServices `json:"loadBalancerServices,omitempty"`
+	// NodeToleration contains optional settings for default tolerations.
+	// +optional
+	NodeToleration *SettingNodeToleration `json:"nodeToleration,omitempty"`
```

Comment:
Why is this configurable in the `Garden` resource while it is not configurable in the `Seed` resource (here, we use the gardenlet component config)? Why don't we use the operator component config here as well to keep this harmonized?